functions.md

```diff
@@ -120,12 +134,39 @@ const squareNum = (num) => {
 
 We can refactor the function to:
 
-```js    
-const squareNum = num => num * num;
+```js
+const squareNum = (num) => num * num;
 ```
 
 Notice the following changes:
 
 - The parentheses around num have been removed, since it has a single parameter.
 - The curly braces { } have been removed since the function consists of a single-line block.
-- The return keyword has been removed since the function consists of a single-line block.
\ No newline at end of file
+- The return keyword has been removed since the function consists of a single-line block.
+
+A typical example of function and it's implementation.
+
+```js
+let factorial = (n) => {
+  let num = 1;
+  /*
+   for (let i = 2; i <= n; i++) {
+    num = num * i
+   }
+   */
```

Comment:
Is there a good reason for using a commented code? If not, kindly delete.

Reply:
👍